index.html

```diff
@@ -1213,59 +1216,94 @@
       <h2>
         Examples
       </h2>
-      <h3>
-        Playback of depth and color streams from same device group.
-      </h3>
-      <pre class="example">
-navigator.mediaDevices.getUserMedia({
-  video: {videoKind: {exact: "color"}, groupId: {exact: id}}
-}).then(function (stream) {
-    // Wire the media stream into a &lt;video&gt; element for playback.
-    // The RGB video is rendered.
-    var video = document.querySelector('#video');
-    video.srcObject = stream;
-    video.play();
+      <aside title="Playback of depth and color streams from same device group"
+      class="example">
+        <pre class="highlight">
+async function attachVideoStream(el, kind, groupId) {
+  const constraints = {
+    video: {
+      mandatory: {
+        groupId: groupId
+      },
+      videoKind: { exact: kind }
+    }
   }
-);
 
-navigator.mediaDevices.getUserMedia({
-  video: {videoKind: {exact: "depth"}, groupId: {exact: id}}
-}).then(function (stream) {
-    // Wire the depth-only stream into another &lt;video&gt; element for playback.
-    // The depth information is rendered in its grayscale representation.
-    var depthVideo = document.querySelector('#depthVideo');
-    depthVideo.srcObject = stream;
-    depthVideo.play();
-  }
-);
+  const stream = await navigator.mediaDevices.getUserMedia(constraints);
```

Comment:
I like await. Is it a problem that it looks different compared to original approach in main spec?
https://w3c.github.io/mediacapture-main/#examples